src/clj/ataru/forms/form_access_control.clj

```diff
@@ -162,6 +164,39 @@
       (log/info (str "Saving updated form " form-key ", changed field id " old-field-id " to " new-field-id))
       (post-form updated-form session tarjonta-service organization-service audit-logger))))
 
+(defn- update-payment-module-to-form
+  [form session audit-logger]
+  (let [sections (:content form)
+        payment-section (kk-application-payment-module)
+        updated-content (map #((if (= (:id %) kk-application-payment-wrapper-key)
+                                 payment-section
+                                 %)) sections)
+        updated-form (assoc form :content updated-content)]
+    (log/info "updating kk-application-payment-module to form " (:key form) " with id " (:id form))
+    (form-store/create-form-or-increment-version! updated-form session audit-logger)
+    "Lomakkeen maksumoduuli päivitetty"))
+
+(defn- add-payment-module-to-form
+  [form session audit-logger]
+  (let [sections (:content form)
+        payment-section (kk-application-payment-module)
+        updated-content (concat (take 2 sections) [payment-section] (drop 2 sections))
```

Comment:
Tähän voisi ehkä lyhyen kommentin lisätä, että miksi tehdään näin. Onko mahdollista että oletussijoittelu muuttuisi joskus?